docker/Dockerfile

```diff
@@ -162,11 +162,13 @@ COPY constants.xml .
 COPY constants_local.xml .
 COPY build.sh .
 COPY CMakeLists.txt .
+COPY tests/ tests
+COPY isolated-server-accounts.json .
 
 # Build & install Zilliqa
 RUN --mount=type=cache,target=/ccache/ \
     --mount=type=cache,target=/root/.cache/vcpkg/ \
-    CCACHE_BASEDIR="/zilliqa" ./build.sh ninja && cmake --build "/zilliqa/build" --target install
+    CCACHE_BASEDIR="/zilliqa" ./build.sh ninja debug tests && cmake --build "/zilliqa/build" --target install
```

Comment:
We can make this optional in a follow up PR to avoid slowing down normal builds cc @yaron-zilliqa